### ISSUE TITLE
May I know how to get the test image with ground truth label?

### ISSUE THREAD

**Issue by tonytonglt** [koyeongmin/PINet#26]
Very appreciate with your good work. I tested with your model and get the single tested image. May I know how can I get the single image with ground truth label just as you did in your paper? I wound like to visualize and compare the result between ground truth and the tested one. Thank you very much.

**Comment by koyeongmin** [comment]
tuSimple dataset provide their ground truth of the testing set by the same format to training set. You can utilize data loader.
